src/commands/daily.ts

```
@@ -9,8 +9,8 @@ import {
 import { formatDateCompact, getDefaultClaudePath, loadDailyUsageData } from '../data-loader.ts';
 import { detectMismatches, printMismatchReport } from '../debug.ts';
 import { log, logger } from '../logger.ts';
-import { sharedCommandConfig } from '../shared-args.internal.ts';
-import { formatCurrency, formatModelsDisplay, formatNumber, pushBreakdownRows, ResponsiveTable } from '../utils.internal.ts';
+import { sharedCommandConfig } from '../_shared-args.ts';
```

ESLint

[error] Expected "../_shared-args.ts" to come before "../logger.ts".

(perfectionist/sort-imports)